.github/workflows/build_nightly.yml

```
@@ -0,0 +1,152 @@
+name: build_nightly
+
+on:
+  push:
+    branches:
+      - develop
+  workflow_dispatch:
+    inputs:
+      build_ios:
+        type: boolean
+        description: iOS
+        default: true
+      build_android:
+        type: boolean
+        description: Android
+        default: true
+      build_web:
+        type: boolean
+        description: Web
+        default: true
+
+defaults:
+  run:
+    working-directory: sample_app
+
+env:
+  FLUTTER_VERSION: "3.29.0"
```

Also, we can use "3.x" here instead, It will automatically use the latest version in the 3.x range.
```suggestion
  FLUTTER_VERSION: "3.x"
```